auth/app/src/main/java/com/google/firebase/quickstart/auth/java/ChooserFragment.java

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Since this is a new file, we should probably take the opportunity to update the year on the license

```suggestion
 * Copyright 2021 Google Inc. All Rights Reserved.
```